examples/distributed_debate/README.md

```diff
@@ -1,54 +1,74 @@
-# Distributed debate competition
+# Distributed Debate Competition
 
-This example simulate a debate competition with three participant agents, including the affirmative side (**Pro**), the negative side (**Con**), and the adjudicator (**Judge**).
-**You can join in the debate as Pro or Con or both.**
+This example demonstrates:
+- How to simulate a debate competition with three participant agents
+- How to allow human participation in the debate
 
-Pro believes that AGI can be achieved using the GPT model framework, while Con contests it.
-Judge listens to both sides' arguments and provides an analytical judgment on which side presented a more compelling and reasonable case.
+## Background
 
-Each agent is an independent process and can run on different machines.
-You can join the debate as Pro or Con by providing the `--is-human` argument.
-Messages generated by any agents can be observed by other agents in the debate.
+This example simulates a debate competition with three participant agents:
+1. The affirmative side (**Pro**)
+2. The negative side (**Con**)
+3. The adjudicator (**Judge**)
 
-Before running the example, please install the distributed version of AgentScope, fill in your model configuration correctly in `configs/model_configs.json`, and modify the `model_config_name` field in `configs/debate_agent_configs.json` accordingly.
+The debate topic is whether AGI can be achieved using the GPT model framework. Pro argues in favor, while Con contests it. Judge listens to both sides' arguments and provides an analytical judgment on which side presented a more compelling case.
 
-### Step 1: setup Pro, Con agent servers
+Each agent is an independent process and can run on different machines. Human participants can join as Pro or Con.
 
+## Tested Models
+
+These models are tested in this example. For other models, some modifications may be needed.
+- Ollama Chat (qwen2:1.5b)
+- Dashscope Chat (qwen-Max)
+- Gemini Chat (gemini-pro)
+
+## Prerequisites
+
+Before running the example:
+- Install the distributed version of AgentScope
+- Fill in your model configuration correctly in `configs/model_configs.json`
+- Modify the `model_config_name` field in `configs/debate_agent_configs.json` accordingly
+- Ensure the specified ports are available and IP addresses are accessible
+
+## Setup and Execution
+
+### Step 1: Setup Pro and Con agent servers
+
+For an LLM-based Pro:
 ```shell
 cd examples/distributed_debate
-# setup LLM-based Pro
 python distributed_debate.py --role pro --pro-host localhost --pro-port 12011
-# or join the debate as Pro by yourself
-# python distributed_debate.py --role pro --pro-host localhost --pro-port 12011 --is-human
 ```
 
+For human participation as Pro:
```

Comment:
Indicate that it is a alternative to the LLM-base, not sequential

Reply:
Done.